Source/Common/CommonBase.h

```diff
@@ -11,6 +11,8 @@ using namespace std;
 #include "flat.h"
 #include "mpi.h"
 
+#define SNDBUFSIZ 1000000000      // MPI immediate message send buffer size

```

Comment:
Makes this buffer size accessible to the Mothership's queueing mechanism.

Reply:
Just thinking out of the box, could this be a build-time-overwiteable-define? gives a bit more flexibility.

Reply:
Why not? As of `86281c51bce09489af6dc26c6237a12d19f31bd9`, try adding `-DSNDBUFSIZ 5` to `CXXFLAGS` in the Makefile.

Add a parameterisable build system to the list of things to do...

Reply:
Honestly, I don't like using preprocessor defines in lieu of a configuration file in a project of this size anyway! The only times I'd use it are:

 - When the optimiser greatly benefits from knowing the value at compile time.

 - When time is short

In the former case, prefixing variables with `CONFIG` and `DEFAULT` makes it clear at a glance that a certain definition is configurable, as opposed to being a straight-up constant that you've defined to satisfy DRY.

Reply:
Cool, so the long-term plan would be config file? happy with that